src/handlers/display-data.js

```diff
@@ -65,16 +65,14 @@ module.exports = async (request, submission) => {
   const activities = (await activitiesApi.getFromLink(request, submission._links.activities.href))
     .sort(activitiesApi.sort)
 
-  // Add a count to the activities
-  activities.map(a => { a.count = 0 })
-
```

Comment:
You're more familiar with the code than I am, but are you absolutely 100% certain that this isn't used? I had a quick look and I wondered if it might be used around lines 86-89:
```
  const allChildren = await smallCatchesApi.getAllChildren(request, activities, '_links.smallCatches.href')

  // Need to show the unknown method if set by the administrator
  const foundInternal = !!allChildren.find(c => c.counts.find(c2 => c2.internal && c2.count))
```
I'm not completely confident of the tests in RCR Frontend, so they might not catch you if you've removed something important.

I don't like the use of `.map`, if it has to stay it'd probably be a bit more obvious to do
```
for (const activity of activities) {
  activity.count = 0
}
```

Reply:
When `activities.map(a => { a.count = 0 })` is run. It creates a new array, it doesn't change the values of the activities array. Below is an example of this:
![image](https://user-images.githubusercontent.com/73947018/100116490-b64fe600-2e6b-11eb-9516-3319bd2a698e.png)

Using this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

So the map is done, but nothing is done with the array after